pkg/auth/multiauthenticator_test.go

```diff
@@ -88,20 +110,89 @@ func TestMultiAuthenticatorWithClientID_AuthenticateToken_Ok(t *testing.T) {
 		Roles:    []string{"tester"},
 	}, &Signature{
 		Type: jwt.SigningMethodHS256,
-		Key:  []byte("my-secret-1"),
+		Key:  []byte("my-secret-2"),
 	}, &Options{
 		Expiration: time.Now().Add(time.Minute * 10).Unix(),
 	})
 	assert.NoError(t, err)
-	assert.NotEmpty(t, rawToken)
+	assert.NotEmpty(t, rawToken1)
 
 	// When.
-	authenticateClaims, err := m.AuthenticateToken(context.Background(), rawToken)
+	authenticateClaims, err := m.AuthenticateToken(context.Background(), rawToken1)
 
 	// Then.
 	assert.NoError(t, err)
 	assert.Equal(t, "issuer-1", authenticateClaims.Issuer)
 	assert.Equal(t, "my@email.com", authenticateClaims.Email)
 	assert.Equal(t, "my-name", authenticateClaims.Name)
 	assert.Equal(t, []string{"tester"}, authenticateClaims.Roles)
+
+	rawToken2, err := Token(&Claims{
```

Comment:
rawToken2 looks the same as rawToken1. What are we testing here?

Reply:
The signature key is different `my-secret-2`. Basically trying to test both authenticators from the same issuer are being used.

Reply:
so, should rawToken1 use "my-secret-1" on line 106? It says "my-secret-2".